test/cloudant.test.js

```diff
@@ -47,9 +47,13 @@ describe('cloudant connector', function() {
         ],
       },
     });
-    Product.destroyAll(function(err) {
-      CustomerSimple.destroyAll(function(err) {
-        done();
+
+    db.automigrate(function(err) {
```

Comment:
I would prefer that we make this change here, since _this only exists in a state that needs fixing_ because of this PR!
Let's continue naming these anonymous functions under test; it's a good pattern IMO.